.github/workflows/docs.yml

```diff
@@ -38,5 +38,5 @@ jobs:
       id-token: write
 
     with:
-      bazel-target: "//docs:github_pages__release"
+      bazel-target: "//docs:incremental_release -- --github_user=${{ github.repository_owner }} --github_repo=${{ github.event.repository.name }}"
```

Comment:
The parameters belong into cicd-workflows, don't they? As they are generic, we don't need to force users to write them out.